### ISSUE TITLE
When item is selected in selectOneMenu in dataTable, value is submitted only after focus change by next user action

### ISSUE THREAD

**Issue by bwhpers** [primefaces/primefaces#3493]
As of today, the issue can be seen in the showcase [https://www.primefaces.org/showcase/ui/data/datatable/edit.xhtml](https://www.primefaces.org/showcase/ui/data/datatable/edit.xhtml )

Try the second table ("Cell Editing with Click and RightClick"), select "Brand". The cell will remain in edit mode until the user triggers a change of focus through another action (i.e. select another cell or just click anywhere else). 

In 5.2 and 6.1 the value is submitted and edit mode exited immediately after item select.

I don't provide an example as it can been seen in the showcase.

**Comment by bwhpers** [comment]
Is this going to be fixed? Is there a workaround?

**Comment by tandraschko** [comment]
As this issue tracker is for the community, you can just provide a PR.
Otherwise you can get a fix via PrimeFaces PRO or wait until someone fixes it.

**Comment by tandraschko** [comment]
Currently the behavior is "by design". 

I will add the "won't fix" label for now until it's completely discussed.

**Comment by bwhpers** [comment]
The behaviour was different up to 6.1. A combo box value in a table cell editor had always been submitted immediately after selection, not on focus change. 
Don't understand the logic, why it has been changed if not inadvertently. 
At least there should be an option (an attribute) to restore the previous behaviour.

**Comment by melloware** [comment]
I agree with Thomas this is the correct behavior and the old behavior was incorrect.
